components/table/table-wrapper.js

```diff
@@ -574,13 +579,35 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 		this._handleTableChange();
 	}
 
-	async _handleTableChange() {
+	async _handleTableChange(mutationRecords) {
+		const flag = window.D2L?.LP?.Web?.UI?.Flags.Flag('table-update-filter-GAUD-6955', true) ?? true;
+		const updates = { count: true, classNames: true, sticky: true, syncWidths: true };
+		if (flag && mutationRecords) {
```

Comment:
Not actually requesting a change, but I've found it easier to have the FF in a separate if block, I feel like it lowers cognitive load in the future when you need to remove the FF. Like:

```
if(flag) {
  if(mutationRecord) {
...
  }
}
```

That way you don't accidentally remove the if(mutationRecords) aspect.

Just a thing I noticed. YMMV